RecoLocalFastTime/FTLClusterizer/interface/MTDCPEFromSiPMTimeBTL.h

```diff
@@ -1,15 +1,15 @@
-#ifndef RecoLocalFastTime_FTLClusterizer_MTDCPEFromSiPMTime_H
-#define RecoLocalFastTime_FTLClusterizer_MTDCPEFromSiPMTime_H 1
+#ifndef RecoLocalFastTime_FTLClusterizer_MTDCPEFromSiPMTimeBTL_H
+#define RecoLocalFastTime_FTLClusterizer_MTDCPEFromSiPMTimeBTL_H 1
 
 //-----------------------------------------------------------------------------
-// \class        MTDCPEFromSiPMTime
+// \class        MTDCPEFromSiPMTimeBTL
 //-----------------------------------------------------------------------------
 
 #include "RecoLocalFastTime/FTLClusterizer/interface/MTDCPEBase.h"
 
-class MTDCPEFromSiPMTime : public MTDCPEBase {
+class MTDCPEFromSiPMTimeBTL : public MTDCPEBase {
```

Comment:
now this looks like it serves only BTL.
Perhaps ```MTDCPESiPMTimeBTLGenericETL``` is more clear?

Reply:
Hi,
It seems that the name of the class is relative to the next question you asked, e.g. if `MTDCPEBase` (better called `MTDCPEGeneric`) should still be available.
So I think maybe we can wait to see whether there is someone that can comment on that question and adjust the name of this class based the comments given by other experts.
Do you think that feasible?

Thanks.

Reply:
It's not necessarily connected to the name of what is now ```MTDCPEBase```. The current name confusingly sounds like it's a CPE for just BTL.